mycro.sh

```diff
@@ -17,7 +33,7 @@ Restart the server using
 Stop the server using
 ./mycro.sh stop
 
-Navigate your browser to localhost:3000 
+Navigate your browser to app.mycrocoin.aq
```

Comment:
.aq?

Reply:
.aq is the TLD for antarctica. I seem to recall there's certain use cases where it's handy to use a real TLD in dev. AQ is reserved for organizations working in antartica so I don't think anyone is going to register this any time soon (unless we migrate our headquarters very far south)

Reply:
done